core/src/main/scripts/migrate_db.py

```diff
@@ -209,6 +210,42 @@ def check_reference_genome(portal_properties, cursor, force_migration):
         if not force_migration:
             sys.exit(1)
 
+def check_and_exit_if_fusions(cursor):
+    try:
+        # look for any fusion records
+        # TODO is this OK or maybe we could have fusions from mutation files?
+        cursor.execute(
+            """
+                SELECT COUNT(*)
+                FROM mutation_event
+                WHERE MUTATION_TYPE = "Fusion";
+            """)
+        fusion_count = cursor.fetchone()
+        if (fusion_count[0] >= 1):
+            print('Found %i fusions in the mutation_event table.  Fusions are being replaced by structural variants. Any study with fusions need to be dropped.' % (fusion_count), file=ERROR_FILE)
```

Comment:
Tune message : Maybe "Found %i records in the mutation_event table where the mutation_type? was "Fusion". The latest database schema does not allow records in the mutation table where mutation_type? is set to "Fusion". Studies linked to existing records of this type should be deleted in order to ..."